src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -4067,12 +4067,12 @@ void ObjectMgr::LoadPlayerInfo()
     {
         uint32 oldMSTime = getMSTime();
 
-        //                                                0      1      2       3
-        QueryResult result  = WorldDatabase.Query("SELECT class, level, basehp, basemana FROM player_classlevelstats");
+        //                                                  0      1      2        3
+        QueryResult result  = WorldDatabase.Query("SELECT Class, Level, BaseHP, BaseMana FROM player_class_stats");
```

Comment:
Okay, but why not initialize this at L#4186 with the rest of the player_class_stats stuff?

Reply:
Don't be lazy, let's not query the same table twice when we don't need to :D